psd-web/Gemfile

```
@@ -65,6 +65,7 @@ group :development, :test do
   gem "rubocop"
   gem "rubocop-govuk", "~> 2.0.0"
   gem "rubocop-performance"
+  gem "rubocop-rspec", require: false
```

I think it’d be good to start specifying versions in the Gemfile using the pessimistic operator (`~>`) - apparently called the “twiddle-wakka”:

```suggestion
  gem "rubocop-rspec", "~> 1.37", require: false
```